src/main/java/com/gregtechceu/gtceu/api/blockentity/PipeBlockEntity.java

```diff
@@ -437,10 +432,4 @@ public static boolean isFaceBlocked(int blockedConnections, Direction side) {
     public static boolean isConnected(int connections, Direction side) {
         return (connections & (1 << side.ordinal())) > 0;
     }
-
-    @Override
-    public void load(CompoundTag tag) {
-        TagFixer.fixFluidTags(tag);
-        super.load(tag);
-    }
```

Comment:
Why was this code removed, and why was anything in this class changed at all?

Reply:
The fluid tag fix call doesn't do anything when called directly on a block entity.

Some of the Pipe BE code was changed slightly as I unified some util methods and interface members which were repeated across the codebase into the `IGregtechBlockEntity` interface